MinecraftClient/Protocol/Handlers/PacketTypesIn.cs

```diff
@@ -132,6 +132,11 @@ public enum PacketTypesIn
         ChatPreview,
         ServerData,
         SetDisplayChatPreview,
-        SystemChat
+        SystemChat,
+
+        // Added in 1.19.1 (1.19.2)
+        ChatSuggestions,
+        HideMessage,
+        MessageHeader
```

Comment:
I'd suggest to order the items alphabetically, and, if you want, put a comment on the right stating in which version the packet was introduced.

Reply:
Okay, sorted them.